### ISSUE TITLE
Export Final Results of FF OH Pad Deliniator as a Shapefile

### ISSUE THREAD

**Issue by ezys** [SkyTruth/CrowdProjects#33]
Please produce a shapefile of the pad delineator results with 50% agreement.

**Comment by Yolandeezy** [comment]
@ezys I have two shapefiles for you: 1) The reduced pads, by 50% agreement, that the pad-reducer.py script produced. Because some of the sites had more than one task_id, they ended having more than one final polygon so I had to do a dissolve. 2) The results of the aforementioned dissolved shapefile, which now has only one polygon at each pad.  

I made sure that each polygon has the appropriate task_id after the dissolve, explode routine. However, this means that each individual polygon is it's own feature. Adjacent polygons originating from the same task_id are no longer one feature. 
If you would like a layer that has one feature per task_id, even if there are multiple pads at that site, there is a way to join the features by task_id. However, I'm not sure how to do that so it would take me a little time to get that to you. 

Meanwhile, I will give you the second shapefile mentioned.
